From 071b8936386b0b44475c91511d85479e5c633bc5 Mon Sep 17 00:00:00 2001
From: Kai Koehne <kai.koehne@theqtcompany.com>
Date: Thu, 11 Dec 2014 13:54:23 +0100
Subject: [PATCH] ANGLE: Do not use std::strlen

This is a cherry-pick from upstream change

e7cfb3dd2029c1bfe5c175ad994c03cac221ad4d

Change-Id: Iefe01545319f9ad268c0c6bf8e8b2181e09d8a84
---
 src/3rdparty/angle/src/libGLESv2/Shader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/3rdparty/angle/src/libGLESv2/Shader.cpp b/src/3rdparty/angle/src/libGLESv2/Shader.cpp
index 5bca746..024ef8f 100644
--- a/src/3rdparty/angle/src/libGLESv2/Shader.cpp
+++ b/src/3rdparty/angle/src/libGLESv2/Shader.cpp
@@ -53,7 +53,7 @@ void Shader::setSource(GLsizei count, const char *const *string, const GLint *le
     {
         if (length == nullptr || length[i] < 0)
         {
-            stream.write(string[i], std::strlen(string[i]));
+            stream.write(string[i], strlen(string[i]));
         }
         else
         {
-- 
1.9.4.msysgit.0

